src/states/adult.rs

```diff
@@ -115,13 +115,17 @@ impl Adult {
         Ok(node)
     }
 
+    pub fn closest_elders_to(&self, name: &XorName) -> impl Iterator<Item = &P2pNode> {
+        self.chain.closest_section_info(*name).1.member_nodes()
```

Comment:
Is that correct?
Probably needs test for this.

I do not think this is correct, the chain as created will still have information as it was at the point of the last Elder change (though it may change). I think this needs to be explicit and return our current elders.

Reply:
I didn't understand this point. Shouldn't we be getting info only from the chain as that's the only view we trust ?

Reply:
My point is we should probably call `self.chain.our_info().member_nodes()`